planb/cli.py

```diff
@@ -78,8 +82,45 @@ def create(regions: list,
     create_cluster(options=locals())
 
 
-sns_topic_help = 'SNS topic name to send Auto-Recovery notifications to'
-sns_email_help = 'Email address to subscribe to Auto-Recovery SNS topic'
+@cli.command()
+@click.option('--region', type=str, required=True)
+@click.option('--cluster-name', type=str, required=True)
+@click.option('--cluster-size', default=3, type=int, help='number of nodes per region, default: 3')
+@click.option('--dc-suffix', type=str, required=True)
+@click.option('--num-tokens', default=256, type=int, help='number of virtual nodes per node, default: 256')
+@click.option('--instance-type', default='t2.medium', help='default: t2.medium')
+@click.option('--volume-type', default='gp2', help='gp2 (default) | io1 | standard')
+@click.option('--volume-size', default=16, type=int, help='in GB, default: 16')
+@click.option('--volume-iops', default=100, type=int, help='for type io1, default: 100')
+@click.option('--no-termination-protection', is_flag=True, default=False)
+@click.option('--use-dmz', is_flag=True, default=False, help='deploy into DMZ subnets using Public IP addresses')
+#@click.option('--hosted-zone', help='create SRV records in this Hosted Zone')
+#@click.option('--scalyr-key')
```

Comment:
This needs more thought.  If you have Scalyr key configured in the old setup, the new nodes will get it as well because we just copy the UserData from one of existing nodes in full.

At the same time one might want to provided a different key for the new nodes, so we can allow this to override the value in UserData.

For the Route53 record, given that this is "extend" we could add the new nodes to an existing record.  But that might not work with all client configurations.  A safer option is to create a separate record, adding the DC suffix, for example.  Then already deployed clients are unaffected, but using the new ring requires changing the deployment config slightly.  These are the trade-offs.